Sources/PackageDescription/ReleaseAction.swift

```diff
@@ -0,0 +1,75 @@
+/// Describes individual release action
+public struct ReleaseAction: Equatable, Codable {    
+    /// Release action order
+    public let order: Order
+    
+    /// Action to run
+    public let action: Action
+    
+    init(order: Order,
+         action: Action) {
+        self.order = order
+        self.action = action
+    }
+    
+    /// Creates custom pre action
+    /// - Parameters:
+    ///     - tool: Name of tool you want to run
+    ///     - arguments: Arguments to pass to the tool
+    public static func pre(tool: String,
+                           arguments: [String] = []) -> ReleaseAction {
+        return ReleaseAction(order: .pre,
+                             action: .custom(tool: tool, arguments: arguments))
+    }
+    
+    /// Creates custom post action
+    /// - Parameters:
+    ///     - tool: Name of tool you want to run
+    ///     - arguments: Arguments to pass to the tool
+    public static func post(tool: String,
+                            arguments: [String] = []) -> ReleaseAction {
+        return ReleaseAction(order: .post,
+                            action: .custom(tool: tool, arguments: arguments))
+    }
+    
+    /// Creates predefined pre action
+    /// - Parameters:
+    ///     - predefinedAction: Specify which `PredefinedAction` to run
+    public static func pre(_ predefinedAction: PredefinedAction) -> ReleaseAction {
+        return releaseAction(predefinedAction,
+                             order: .pre)
+    }
+    
+    /// Creates predefined post action
+    /// - Parameters:
+    ///     - predefinedAction: Specify which `PredefinedAction` to run
+    public static func post(_ predefinedAction: PredefinedAction) -> ReleaseAction {
+        return releaseAction(predefinedAction,
+                             order: .post)
+    }
+    
+    static func releaseAction(_ predefinedAction: PredefinedAction, order: Order) -> ReleaseAction {
```

Comment:
❓ Is this intentionally internal?

Reply:
Yes, it is, user is supposed to use these two public-facing methods:
```swift
public static func pre(_ predefinedAction: PredefinedAction) -> ReleaseAction
public static func post(_ predefinedAction: PredefinedAction) -> ReleaseAction
```
Which then translate to this method, which is just a convenience method, though.